static/app/components/group/groupSummary.tsx

```diff
@@ -108,7 +108,9 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
               <HeadlinePreview>{data.headline}</HeadlinePreview>
               <SummaryPreview
                 dangerouslySetInnerHTML={{
-                  __html: singleLineRenderer(`Details: ${data.summary}`),
+                  __html: singleLineRenderer(
+                    `Details: ${data.summary.replaceAll('\n', ' ').replaceAll('-', '')}`
```

Comment:
Hmm, why are we removing new lines and dashes?

Reply:
Because bulleted markdown contains \n and -, and if you don't remove it, the markdown rendering turns multiline when the summary panel is collapsed, even with the `singleLineRenderer`